spark/v3.1/spark/src/main/java/com/netease/arctic/spark/SparkSQLProperties.java

```diff
@@ -27,4 +27,8 @@ public class SparkSQLProperties {
 
   public static final String USE_TIMESTAMP_WITHOUT_TIME_ZONE_IN_NEW_TABLES_DEFAULT =
           "false";
+
+  public static final String FORCE_REFRESH = "force.refresh";
```

Comment:
Should we start the property name with `spark.sql.arctic`?
How about rename the property name to `spark.sql.arctic.refresh-catalog-before-usage`?

Reply:
BTW, You may need to set this property to `true` in `LocalTerminalSession` in AMS.